### DISCUSSION TITLE
How to know on the browser if the ENV is dev or prod?

### DISCUSSION THREAD

**Discussion by omarryhan** [vercel/next.js#13325] [Help] [answered]
I know how to do that on the server side with Node JS (with `process.env.NODE_ENV`), but not sure how this can be achieved on the browser. For context, I need to know this to be able to set my Google Analytics plugin to either `debug` in dev or `!debug` in prod. Any help?

**Comment by omarryhan** [comment, marked as answer]
Whoops, nvm: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables

**Comment by Timer** [comment]
`process.env.NODE_ENV` also works on the client-side.

**Reply by omarryhan** [reply]
Awesome, thanks.
